packages/quantic/force-app/examples/main/lwc/actionAddCustomSortOptions/actionAddCustomSortOptions.js

```diff
@@ -0,0 +1,19 @@
+import {api, LightningElement} from 'lwc';
+
+export default class ActionAddCustomSortOptions extends LightningElement {
+  @api engineId;
+  @api disabled;
+  @api withInvalidOptions = false;
+  @api label;
+
+  handleAddCustomSortOptions() {
+    const eventName = this.withInvalidOptions
+      ? 'addInvalidCustomSortOptions'
+      : 'addCustomSortOptions';
```

Comment:
instead of playing with the name of the events, always send and event called `addCustomSortOptions` and include a boolean in its details: 
```
{
   invlidSortOptions: false;
}
```
Dealing with a boolean is better than dealing with two magic strings as event names.

Reply:
yeah thats a good idea, we can also probably remove the whole 

  interface SortActionsOptions {
    withCustomSortOptions?: boolean;
    withInvalidCustomSortOptions?: boolean;
  }